### ISSUE TITLE
BBC: 9fdd7fc479 caused (masked) test failures in Data-Alias

### ISSUE THREAD

**Issue by jkeenan** [Perl/perl5#20572]
The most recent release of CPAN distribution Data-Alias (version 1.27) is experiencing test failures.  These test failures would have been apparent several months ago, but Data::Alias was failing to compile.  Now that those compilation problems have been overcome, the test failures have become apparent.  Sample CPANtesters report: http://www.cpantesters.org/cpan/report/7d8ea5da-7244-11ed-8a8f-377406b3d53a.

Bisecting with this invocation ...
```
perl Porting/bisect.pl \
--module=Data::Alias \
--start=v5.37.2 \
--end=v5.37.3
```
... points to this commit:
```
commit 9fdd7fc4796d89d16dceea42f2af91e4fde296ed
Author:     Richard Leach <rich+perl@hyphen-dash-hyphen.info>
AuthorDate: Fri Jul 8 14:52:19 2022
Commit:     Richard Leach <rich+perl@hyphen-dash-hyphen.info>
CommitDate: Wed Aug 17 10:19:10 2022

    Implement OP_PADSV_STORE - combined sassign/padsv OP
```
Data-Alias's maintainer is already aware of this problem, having filed https://rt.cpan.org/Ticket/Display.html?id=145279.  This is a case where a CPAN distribution has to make some adaptations to keep up with blead.  I don't think blead needs any corrective action (though I hope @richardleach takes a look nonetheless), so I'm filing this ticket mainly as a pointer to the RT ticket.

**Comment by richardleach** [comment]
I've had a quick skim, will hopefully get some time at the weekend/over the holidays for a proper look.

**Comment by richardleach** [comment]
I've started looking at this. Coming to Data::Alias cold, it's not a trivial task. Will put progress comments on the RT ticket.

**Comment by mvduin** [comment]
This one has been fixed in git: https://github.com/mvduin/perl-Data-Alias/commit/6e24085ae95bca30f87d02d810659da1869fe8d8

But perl 5.37 introduced more breakage in Data::Alias which I haven't had time to fix yet, which is why I haven't made a new build yet

**Comment by mvduin** [comment]
I've just uploaded Data::Alias 1.28 which fixes compatibility with perl 5.37.3 - 5.37.7

**Comment by jkeenan** [comment]
> I've just uploaded Data::Alias 1.28 which fixes compatibility with perl 5.37.3 - 5.37.7

I tested this yesterday.  LGTM.  Closing this ticket; thanks for your diligence.
